.github/CHANGELOG.md

```diff
@@ -21,6 +21,58 @@ This release contains contributions from (in alphabetical order):
 
 <h3>New features since last release</h3>
 
+* The transform for the Jacobian of the classical preprocessing within a QNode,
+  `qml.transforms.classical_jacobian`, now takes a keyword argument `argnum` to specify
+  the QNode argument indices with respect to which the Jacobian is computed.
+  [(#1645)](https://github.com/PennyLaneAI/pennylane/pull/1645)
+
+  The output behaviour with `argnum=None` differs slightly for different interfaces,
+  following the respective output shape of the native `jacobian` equivalent:
+
+  ==============  =====================  ====================  ========================
+  Interface       ``argnum=None``        ``type(argnum)=int``  ``argnum=Sequence[int]``
+  ==============  =====================  ====================  ========================
+  ``'autograd'``  ``tuple(arrays)`` [1]  ``array``              ``tuple(array)``
+  ``'jax'``       ``array``              ``array``              ``tuple(array)``
+  ``'tf'``        ``tuple(arrays)``      ``array``              ``tuple(array)``
+  ``'torch'``     ``tuple(arrays)``      ``array``              ``tuple(array)``
+  ==============  =====================  ====================  ========================
+
+  [1] If all QNode argument are scalars, the tuple is unpacked and the one-dimensional Jacobian
+  arrays are stacked into one ``array``. If there only is one QNode argument, the tuple is 
+  unpacked as well. Both is due to the behaviour of ``qml.jacobian``.
```

Comment:
Now that the changelog is rendering correctly, I wonder if this is too much technical detail for the changelog.

How would you feel about removing the above table and caption, and just going straight into the code example below?

Afterwards, you could then say

````markdown
For more usage details, please see the
[classical Jacobian docstring](https://pennylane.readthedocs.io/en/latest/code/api/pennylane.transforms.classical_jacobian.html).